frontend/src/scenes/session-recordings/filters/RecordingsUniversalFilters.tsx

```diff
@@ -86,7 +87,7 @@ export const RecordingsUniversalFilters = ({
     useMountedLogic(actionsModel)
     useMountedLogic(groupsModel)
 
-    const durationFilter = filters.duration[0]
+    const durationFilter = filters.duration?.length ? filters.duration[0] : defaultRecordingDurationFilter
```

Comment:
```suggestion
    const durationFilter = filters.duration?.[0] ?? defaultRecordingDurationFilter
```

i'd probably do that to potentially make it easier to read
i guess in a perfect world `filters.duration` would never be invalid but 🙈